sources/osgViewer/Viewer.js

```diff
@@ -272,60 +275,61 @@ MACROUTILS.createPrototypeObject(
 
                 var renderer = this.getCamera().getRenderer();
 
-                if (stats) stats.rStats('cull').start();
+                if (stats) stats.getCounter('cull').start();
 
                 renderer.cull();
 
-                if (stats) stats.rStats('cull').end();
+                if (stats) stats.getCounter('cull').end();
 
                 timerGPU.pollQueries();
                 timerGPU.start('glframe');
 
                 if (stats) {
```

Comment:
Above one liner "if", below "if" with curly braces. Could we have this in a consistent way?